confidant/routes/jwks.py

```diff
@@ -6,11 +6,14 @@
 from confidant.services.jwkmanager import jwk_manager
 from confidant.schema.jwks import jwt_response_schema, JWTResponse, \
     jwks_list_response_schema, JWKSListResponse
-
+from confidant.settings import JWT_MAPPING_MODULE
+from confidant.utils import misc
 
 logger = logging.getLogger(__name__)
 blueprint = blueprints.Blueprint('jwks', __name__)
 
+jwt_mapping_check = misc.load_module(JWT_MAPPING_MODULE)
```

Comment:
let's conform to use `acl_module_check = misc.load_module(settings.ACL_MODULE)` instead.  This centralizes the RBAC logic.